.github/ISSUE_TEMPLATE/config.yaml

```diff
@@ -0,0 +1,11 @@
+blank_issues_enabled: false
```

Comment:
I think this one should be updated

Reply:
you mean it should be set to true? so people can create blank issues if they want to

Reply:
I mean config content should be updated to remove CLI/SDK links

Reply:
got it. I removed them